kubernetes/compose/v1beta1/stack_test.go

```diff
@@ -0,0 +1 @@
+package v1beta1
```

Comment:
This file is here to get representable coverage?

Reply:
Yep exactly.